File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    return eval(binomial)
    binomial = binomial.split(' ')
    if binomial[1] == '+':
        return int(binomial[0]) + int(binomial[2])
    elif binomial[1] == '-':
        return int(binomial[0]) - int(binomial[2])
    elif binomial[1] == '*':
        return int(binomial[0]) * int(binomial[2])
    else:
        return int(binomial[0]) / int(binomial[2])

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        int answer = 0;
        
        String[] bin = binomial.split(" ");
        
        if (bin[1].equals("+")) {
            answer = Integer.parseInt(bin[0]) + Integer.parseInt(bin[2]);
        } else if (bin[1].equals("-")) {
            answer = Integer.parseInt(bin[0]) - Integer.parseInt(bin[2]);
        } else if (bin[1].equals("*")) {
            answer = Integer.parseInt(bin[0]) * Integer.parseInt(bin[2]);
        }
        
        return answer;
    }
}